src/main/java/de/tum/in/www1/artemis/web/rest/ParticipationResource.java

```diff
@@ -283,6 +283,14 @@ public ParticipationResource(ParticipationService participationService,
             if(!courseService.userHasTAPermissions(course)) {
                 return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
             }
+        } else if (!courseService.userHasTAPermissions(course)) {
```

Comment:
After merging the other pull request, this method call has to be renamed, right?